servers/rendering/shader_warnings.cpp

```diff
@@ -73,6 +75,14 @@ String ShaderWarning::get_name() const {
 	return get_name_from_code(code);
 }
 
+Variant ShaderWarning::get_extra_arg1() const {
+	return extra_arg1;
+}
+
+Variant ShaderWarning::get_extra_arg2() const {
+	return extra_arg2;
+}
```

Comment:
Maybe this could be an Array so that warnings can pass an arbitrary numbers of additional parameters?

Reply:
Done, I've also replaced `#if DEBUG_ENABLED` to `#ifdef DEBUG_ENABLED` in a few files (I think its an error)